validator/project_validator_test.go

```diff
@@ -1421,6 +1421,22 @@ func TestValidatePluginCommands(t *testing.T) {
 			So(validatePluginCommands(project), ShouldNotResemble, ValidationErrors{})
 			So(len(validatePluginCommands(project)), ShouldEqual, 1)
 		})
+		Convey("an error should be thrown if a shell.exec command is missing params", func() {
+			project := &model.Project{
+				Functions: map[string]*model.YAMLCommandSet{
```

Comment:
since the ticket originally came out of a struggle with "params" being misspelled, would it maybe be possible to add a test that mimics that so that we know for sure that it covers that use case?